src/crimson/osd/shard_services.cc

```diff
@@ -79,9 +79,10 @@ seastar::future<> ShardServices::dispatch_context_messages(
 {
   auto ret = seastar::parallel_for_each(std::move(ctx.message_map),
     [this](auto& osd_messages) {
-      auto& [peer, messages] = osd_messages;
+      auto &[peer, messages] = osd_messages;
```

Comment:
i am still in favor of putting `&` right next to `auto`. for more examples, see https://google.github.io/styleguide/cppguide.html and https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines .

Reply:
I actually do too...
WIll fix that.